utils/build/docker/set-system-tests-weblog-env.Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM system_tests/weblog
 
 ARG SYSTEM_TESTS_LIBRARY
 ENV SYSTEM_TESTS_LIBRARY=$SYSTEM_TESTS_LIBRARY
+ENV DD_IAST_EXPERIMENTAL_PROPAGATION_ENABLED='true'
```

Comment:
this must be set at scenario level, for the scenario that execute the tested behavior

Reply:
This PR is only to trigger the pipelines and see that everything still fine after enable it. My bad for not advising in the title of the PR that it doesn't need review. When the pipeline finish I'll close it

Reply:
No worry :)